modules/meeting/app/components/meetings/sidebar/details_component.sass

```diff
@@ -0,0 +1,12 @@
+@import 'helpers'
+.op-meeting-sidebar-details-participants
+  display: none
+  &-button
+    padding: 0 !important
+    color: var(--content-link-color)
+
+
+
+@media screen and (max-width: $breakpoint-sm)
+  .op-meeting-sidebar-details-participants
+    display: block !important
```

Comment:
CSS files of components should only contain layouting CSS and things that cannot be done with the `system_arguments`. That means, this whole file should be not needed. You can define the display value depending on the breakpoint like this: `d: : [:none, :block, :none, :none, :none]`. See [here](https://primer.style/view-components/lookbook/pages/system_arguments/) for more information. The color and the padding can also be defined via the system_arguments.